site-src/blog/2023/0801-welcome-gamma.md

```diff
@@ -0,0 +1,127 @@
+requiring users of the Gateway API to relearn everything they understand about
+the Gateway API itself.
+
+Over the last year, GAMMA has dug deeply into the challenges and possible
+solutions around using the Gateway API for service mesh. The end result is a
+small number of [enhancement
+proposals](https://gateway-api.sigs.k8s.io/v1beta1/contributing/gep/) that
+subsume many hours of thought and debate, and provide a minimum viable path to
+allow the Gateway API can be used for service mesh. Of particular note:
+
+- [GEP-1324](https://gateway-api.sigs.k8s.io/geps/gep-1324/) provides an
+  overview of the GAMMA goals and some important definitions. This GEP is well
+  worth a read for its discussion of the problem space.
+- [GEP-1426](https://gateway-api.sigs.k8s.io/geps/gep-1426/) defines how to
+  use Gateway API route resources, such as HTTPRoute, to manage traffic within
+  a service mesh.
+- [GEP-1686](https://gateway-api.sigs.k8s.io/geps/gep-1686/) builds on the
+  work of [GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/) to define
+  a _conformance profile_ for service meshes to be declared conformant with
+  the Gateway API.
+
+### How will mesh routing work when using the Gateway API?
+
+All the details are in
+[GEP-1426](https://gateway-api.sigs.k8s.io/geps/gep-1426/), but the short
+version for Gateway API 0.8.0 is that an HTTPRoute can now have a `parentRef`
+that is a Service, rather than just a Gateway. We anticipate future GEPs in
+this area as we gain more experience with service mesh use cases -- binding to
+a Service makes it possible to use the Gateway API with a service mesh, but
+there are several interesting use cases that remain difficult to cover.
+
+### How does Gateway API conformance work for a service mesh?
+
+One of the challenges that the GAMMA initiative ran into is that Gateway API
+conformance was strongly tied to the idea that a given implementation provides
+an ingress controller. Many service meshes don't, and requiring a
+GAMMA-conformant mesh to also implement an ingress controller seemed
+impractical at best. This resulted in work restarting on Gateway API
+_conformance profiles_, as discussed in
+[GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/).
+
+The basic idea of conformance profiles is that we can define subsets of the
+Gateway API, and allow implementations to choose - and document! - which
+subsets they conform to. GAMMA is adding a new profile, named `Mesh`, which
+checks only the mesh functionality as defined by GAMMA; SIG-Network-Policy is
+going to be using this concept as well.
+
+## What else is in Gateway API 0.8.0?
+
+In addition to GAMMA becoming experimental, Gateway API 0.8.0 also includes a
+way to configure routing scope for a given Gateway (see
+[GEP-1651](https://gateway-api.sigs.k8s.io/geps/gep-1651/) and updates the
+HTTPRoute resource to include native configuration for timeouts (see
+[GEP-1742](https://gateway-api.sigs.k8s.io/geps/gep-1742/).
```

Comment:
Just noticed a missing `)` at the end of the line.

Reply:
🤦‍♂️ Thanks for the catch!